new/LIT6281Neihabeya.xml

```diff
@@ -0,0 +1,70 @@
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ንዒ፡ ኀቤየ፡ ኦድንግል፡ ምስለ፡ ወልድኪ፡ ፍቁርኪ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Nǝʿi ḫabeya ʾo-dǝngǝl mǝsla waldǝki fǝqurki</title>
+                <title xml:lang="en" corresp="#t1">Come to me, o virgin, with your beloved son</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2020-12-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethf34"/>.</note>
+                    <ab>
+                        ንዒ፡ ኀቤየ፡ ኦድንግል፡ ምስለ፡ ወልድኪ፡ ፍቁርኪ፡ ኢየሱስ፡ ክርስቶስ፡ ከመ፡ ትባርኪ፡ ላዕሌነ። ንዒ፡ ኀቤየ፡
+                        ኦድንግል፡ ምስለ፡ አዳም፡ ወአቤል፡ ሴት፡ ወሄኖስ፡ ቅዱሳን፡ አበዊሆሙ፡ ለቀደምት፡ እደው፡ ስሙያን።
+                        </ab>
+                    <div type="textpart" subtype="explicit" xml:lang="gez">
+                        <note>This explicit is taken from <ref type="mss" corresp="BDLaethf34"/>.</note>
+                        <ab>ተፈሥሒ፡ ኦሙሐዛ፡ ፍሥሐ፡ ቡርክት፡ አንቲ፡ ወቡሩክ፡ ፍሬ፡ ከርሥኪ፡ እግዚኦ፡ መሐረነ፡ ክርስቶስ፡ ፲ወ፪፡ በል፡ ባርከነ፡ እግዚኦ፡ አምኅክነ፡
+                            ውስተ፡ ኑኃ፡ ሰማይ፡ ተወከፍ፡ ጸሎተነ፡ መሐረነ፡ እግዚኦ፡ ወተሠሃለነ። ነዓኵተከ፡ ወጸሎተ፡ ሃይማኖት፡ በል፡ ወድኅሬሁ፡ ነቢያት፡ በል፡</ab>
+                    </div>
+                </div>
```

Comment:
This prayer seems to be included in the Saatat. I just send you the pdf via Email. In case, it is the same as the one starting from page 12 onwards, you could add the bibliographic reference and this relation:

 ```
<listRelation>
                    <relation name="saws:formsPartOf" active="LIT6281Neihabeya"
                        passive="LIT1961Mashaf"/>
                                    </listRelation>
```

Reply:
Great! Thank you! ;)